ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-commands.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+env > ${SHARED_DIR}/INHERITED_ENVIRONMENT
+ls ${CLUSTER_PROFILE_DIR} > CLUSTER_PROFILE_DIR
+echo ${CLOUD_PROVIDER} > ${SHARED_DIR}/CLOUD_PROVIDER
+# CLOUD_PROVIDER dependent environmental variables
+case "${CLOUD_PROVIDER}" in
+    moc)
+        echo "openstack" > ${SHARED_DIR}/CLUSTER_TYPE
+        echo "external" > ${SHARED_DIR}/OPENSTACK_EXTERNAL_NETWORK
+        echo "openstack-cloud" > ${SHARED_DIR}/OS_CLOUD
+        echo "openstack" > ${SHARED_DIR}/CLUSTER_TYPE
+        echo "m1.s2.xlarge" > ${SHARED_DIR}/OPENSTACK_COMPUTE_FLAVOR
+        ;;
+    vexxhost)
+        echo "openstack" > ${SHARED_DIR}/CLUSTER_TYPE
+        echo "external" > ${SHARED_DIR}/OPENSTACK_EXTERNAL_NETWORK
+        echo "openstack" > ${SHARED_DIR}/OS_CLOUD
+        echo "openstack-vexxhost" > ${SHARED_DIR}/CLUSTER_TYPE
```

Comment:
This overwrites `CLUSTER_TYPE`, is this intended?

Reply:
yes. we have to "CLUSTER_TYPES" depending on which cloud provider we use. 
But I think we can ommit this altogether since I belive the CLUSTER_TYPE will be set by "cluster_profile" under the test declaration. so removing to test that theory.

done